robot/web/web_test.go

```diff
@@ -1178,6 +1178,74 @@ func TestRawClientOperation(t *testing.T) {
 	test.That(t, svc.Close(ctx), test.ShouldBeNil)
 }
 
+func TestUnaryRequestCounter(t *testing.T) {
+	echoAPI := resource.NewAPI("rdk", "component", "echo")
+	resource.RegisterAPI(echoAPI, resource.APIRegistration[resource.Resource]{
```

Comment:
this is a component - if you look at the import, it is `"go.viam.com/api/component/testecho/v1"`

to test service calls, we'll probably have to register a proto that is a viam service, maybe something like the generic service?